notebooks/community/ml_ops/stage1/get_started_vertex_datasets.ipynb

```diff
@@ -8,7 +8,7 @@
       },
```

Comment:
<p>reduce to installs used in this notebook</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/474/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
@andrewferlitsch : I tried to reduce the install and use the default versions.